lib/floe/workflow/runner.rb

```diff
@@ -7,7 +7,7 @@ class Runner
 
       TYPES = %w[docker podman kubernetes].freeze
 
-      def initialize(_options = {})
+      def initialize(_options = {}) # rubocop:disable Style/RedundantInitialize
```

Comment:
> Unnecessary disabling of Style/RedundantInitialize.

Reply:
Rationale behind this, if the child classes are going to define initialize with arguments I like to define the interface at the base class.

Reply:
I understood.

Locally it reports an error here (along with a bunch of "outdated" issues with style)
On the server, it seems ok. confused

Reply:
I removed this rubocop disable